{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--8782aca7-8588-4ca1-94c9-61880eb886d9",
    "created": "2024-08-13T16:04:57.293271Z",
    "modified": "2024-08-13T16:04:57.293275Z",
    "name": "No Title Available",
    "description": "An issue was discovered in glFTPd 2.11a that allows remote attackers to cause a denial of service via exceeding the connection limit.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2021-31645"
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://glftpd.io/"
        },
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://www.exploit-db.com/exploits/49773"
        }
    ]
}